src/Ledger/Utxo.lagda

```diff
@@ -130,8 +130,12 @@ updating this map, which is split into \updateCertDeposits and
 \updateProposalDeposits, responsible for certificates and proposals
 respectively. Both of these functions iterate over the relevant fields
 of the transaction body and insert or remove deposits depending on the
-information seen. Note that some deposits can only be refunded at the
-epoch boundary and are not removed by these functions.
+information seen.  More precisely, \updateCertDeposits is responsible
```

Comment:
There are now two explanations of `updateCertDeposits` and `updateProposalDeposits` here. Please remove this duplication.